script/stress_test.py

```diff
@@ -0,0 +1,418 @@
+        """get latest block number"""
+        start_block, end_block = self.block_range
+        # align block timestamp offset
+        if self.ts_offset is None:
+            if not await self.align_ts_offset(start_block):
+                return None
+
+        if self.last_block is None:
+            return start_block
+        else:
+            # check if next block timestamp is overdue
+            next_block = self.last_block + 1
+            if next_block >= end_block:
+                self.logger.info(f"Block range {self.block_range} finished")
+                if self.running_count == 0:
+                    self.logger.info("All blocks finished, exiting")
+                    exit(0)
+                return None
+            else:
+                block = await self.get_block(next_block)
+                timestamp = int(block["timestamp"], 16) + self.ts_offset
+                current_timestamp = int(time.time())
+                if current_timestamp > timestamp:
+                    return next_block
+                else:
+                    self.logger.info(
+                        f"Block {next_block} timestamp:{timestamp} is not reached, current:{current_timestamp}"
+                    )
+                    await asyncio.sleep(timestamp - current_timestamp)
+                    return None
+
+    async def get_latest_block(self) -> Optional[int]:
+        """get latest block number"""
+        try:
+            response = requests.post(
+                self.eth_rpc,
+                json={
+                    "jsonrpc": "2.0",
+                    "method": "eth_blockNumber",
+                    "params": [],
+                    "id": 1,
+                },
+                timeout=10,
+            )
+            result = response.json()
+            return int(result["result"], 16)
+        except Exception as e:
+            self.logger.error(f"Failed to get latest block: {e}")
+            return None
+
+    def generate_post_data(self, block_number: int) -> Dict[str, Any]:
+        """generate post data"""
+        return {
+            "block_numbers": [[block_number, None]],
+            "block_number": block_number,
+            "prover": "0x70997970C51812dc3A010C7d01b50e0d17dc79C8",
+            "graffiti": "8008500000000000000000000000000000000000000000000000000000000000",
+            "proof_type": self.prove_type,
+            "blob_proof_type": "proof_of_equivalence",
+            "native": {},
+            "risc0": {
+                "bonsai": True,
+                "snark": True,
+                "profile": False,
+                "execution_po2": 20,
+            },
+            "sp1": {"recursion": "plonk", "prover": "network", "verify": True},
+        }
+
+    async def submit_to_raiko(self, block_number: int) -> Optional[str]:
+        """submit block to Raiko"""
+        try:
+            headers = {"x-api-key": "1", "Content-Type": "application/json"}
```

Comment:
What is the purpose of the `x-api-key` header? What content should be filled in actual scenarios, always be "1"?

Reply:
BTW, I didn't see the auth header for [JWT authorization](https://github.com/taikoxyz/raiko/blob/main/host/src/server/api/mod.rs#L56). Doesn't the Raiko HTTP service require this header?

Reply:
no use so far, we will add JWT & https soon.